main-app/directivo/includes/barra-superior-inscripciones-componente.php

```diff
@@ -27,6 +28,7 @@
 $count=0;
 foreach($ordenReal as $clave) {
   $filtroEstado[$count] = [
+    'ID' => $clave,
     'texto' => $estadosSolicitud[$clave],
     'url' => $_SERVER['PHP_SELF']."?estado=".base64_encode($clave)."&curso=".base64_encode($curso)
```

Comment:
Trata de alinear los valores y/o variables que están en lineas seguidas para una mejor legibilidad.

```suggestion
    'ID'      => $clave,
    'texto' => $estadosSolicitud[$clave],
    'url'     => $_SERVER['PHP_SELF']."?estado=".base64_encode($clave)."&curso=".base64_encode($curso)
```

Reply:
ok